File: 백준/Bronze/34691. 대전과학고등학교를 사랑하십니까？/대전과학고등학교를 사랑하십니까？.py
import sys

symbol_map = {
    'animal': 'Panthera tigris',
    'tree': 'Pinus densiflora',
    'flower': 'Forsythia koreana'
}

for line in sys.stdin:
    line = line.strip()
    if line == 'end':
        break
    print(symbol_map[line])

File: 백준/Bronze/34691. 대전과학고등학교를 사랑하십니까？/대전과학고등학교를 사랑하십니까？.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').toString().trim().split('\n');
const symbolMap = {
    'animal': 'Panthera tigris',
    'tree': 'Pinus densiflora',
    'flower': 'Forsythia koreana'
}

for(const line of input){
    if(line === 'end') break;
    console.log(symbolMap[line])
}